FSharpLu.Json/Compact.fs

```diff
@@ -10,18 +13,33 @@ open Microsoft.FSharp.Reflection
 type CompactUnionJsonConverter() =
     inherit Newtonsoft.Json.JsonConverter()
 
+    let processWithContractResolver (serializer:JsonSerializer) (name:string) =
```

Comment:
Now that I think of it, we should not force `toPascal` in the `DefaultContractResolver`. It a DU case is written in all lower letters, it should honor it.